lib/recommendation/ui/pages/add_edit_page.dart

```diff
@@ -90,13 +91,19 @@ class AddEditRecommendationPage extends HookConsumerWidget {
                         final XFile? image =
                             await picker.pickImage(source: ImageSource.gallery);
                         if (image != null) {
-                          if (kIsWeb) {
-                            logo.value = await image.readAsBytes();
-                            logoFile.value = Image.network(image.path);
+                          final size = await image.length();
+                          if (size > maxFileSize) {
+                            displayAdvertToastWithContext(TypeMsg.error,
+                                RecommendationTextConstants.imageSizeTooBig);
```

Comment:
Oh, this is awesome !!
Can you add it to the other modules, or even extract this image picker to the tools ?
I am assigning you an enhancement issue 😉